<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BirdsBudgeting</class>
 <widget class="QMainWindow" name="BirdsBudgeting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BirdsBudgeting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Please enter the following:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RevType">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RevAmt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ExpType">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ExpAmt">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="RevTypeLab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Revenue Type </string>
    </property>
   </widget>
   <widget class="QLabel" name="RevAmtLab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Revenue Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="ExpTypeLab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Expense Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="ExpTypeLab_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Expense Amount</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>631</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Display_2">
       <property name="text">
        <string>Display</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="output"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
